Makefile

```diff
@@ -37,6 +37,11 @@ BAZEL_STARTUP_ARGS := --client_debug $(BAZEL_STARTUP_ARGS)
 BAZEL_BUILD_ARGS := -s $(BAZEL_BUILD_ARGS)
 endif
 
+BUILD_WITH_CONTAINER ?= 0
+ifeq "$(BUILD_WITH_CONTAINER)" "1"
+BAZEL_BUILD_ARGS := $(BAZEL_BUILD_ARGS) --config=docker-clang-libc++
```

Comment:
Where is the magic that defines this config?

Reply:
`.bazelrc`

Reply:
Hehe, but that just makes the executor spawn docker containers to run jobs. Bazel itself is not under docker, right?

Reply:
Correct. Everything else is, though.

Reply:
What I am looking for is `make build_envoy BULD_WITH_CONTAINER=1` will build me an image without having to install anything locally. Building inside the docker container when you could also build locally is adding an orthogonal feature.

Reply:
@mandarjog yes, this will build release images for you, assuming that you have Bazel(isk) installed already.